bats/tests/helpers/vm.bash

```diff
@@ -320,9 +325,51 @@ docker_context_exists() {
     assert_output "$RD_DOCKER_CONTEXT"
 }
 
+RD_VM_OS=
+get_os_type() {
+    [[ -n ${RD_VM_OS:-} ]] || load_var RD_VM_OS || true
+    if [[ -z ${RD_VM_OS:-} ]]; then
+        local id is_rd=false
+        for id in $(rdctl shell cat /etc/os-release | tr -d '"' | awk -F= '/^ID/ { print $2 }'); do
+            case "$id" in
+            opensuse | suse) RD_VM_OS=opensuse ;;
+            alpine) RD_VM_OS=alpine ;;
+            rancher-desktop-wsl-distro) is_rd=true ;;
+            esac
+        done
+        if [[ -z ${RD_VM_OS:-} ]] && $is_rd; then
+            RD_VM_OS=alpine # The existing Alpine-based distribution
+        fi
+        save_var RD_VM_OS
+    fi
+    if [[ -z ${RD_VM_OS:-} ]]; then
+        fail "Failed to detect OS"
+    fi
+    echo "${RD_VM_OS}"
+}
+
+service_control() { # service action
+    local if_started=
+    if [[ ${1:-} == "--ifstarted" ]]; then
+        if_started=$1
+        shift
+    fi
+    local service=$1 action=$2
+    if [[ $(get_os_type) == opensuse ]]; then
```

Comment:
The condition should check for `alpine` because we will never try to support any other non-systemd distro (not that we are likely to have any other systemd distro either, so maybe it doesn't really matter).